src/edit_view.rs

```diff
@@ -59,20 +82,142 @@ const TOP_PAD: f32 = 6.0;
+        _ctx: &mut LayoutCtx) -> LayoutResult
+    {
+        LayoutResult::Size(bc.constrain((0.0, 0.0)))
+    }
+
+    fn mouse(&mut self, event: &MouseEvent, _ctx: &mut HandlerCtx) -> bool { 
+        let MouseEvent { x, y, mods: _, which, count } = *event;
+        if which == MouseButton::Left && count == 1 {
+            let (line, col) = self.xy_to_line_col(x, y);
+            let params = json!({
+                "ty": "point_select",
+                "line": line,
+                "col": col,
+            });
+            self.send_edit_cmd("gesture", &params);
+        }
+        false
+    }
+
+    fn mouse_moved(&mut self, _x: f32, _y: f32, _ctx: &mut HandlerCtx) {}
+
+    fn on_hot_changed(&mut self, _hot: bool, _ctx: &mut HandlerCtx) {}
+
+    fn poke(&mut self, payload: &mut Any, ctx: &mut HandlerCtx) -> bool { 
+        if let Some(cmd) = payload.downcast_ref::<EditViewCommands>() {
+            match cmd {
+                EditViewCommands::ViewId(view_id) => {
+                    self.view_id = view_id.to_string();
+                }
+                EditViewCommands::ApplyUpdate(update) => {
+                    self.apply_update(&update);
+                }
+                EditViewCommands::ScrollTo(line) => {
+                    self.scroll_to(*line);
+                }
+                EditViewCommands::Core(core) => {
+                    self.core = core.clone();
+                }
+                EditViewCommands::Undo => {
+                    self.send_action("undo");
+                }
+                EditViewCommands::Redo => {
+                    self.send_action("redo");
+                }
+                EditViewCommands::UpperCase => {
+                    self.send_action("uppercase");
+                }
+                EditViewCommands::LowerCase => {
+                    self.send_action("lowercase");
+                }
+                EditViewCommands::Transpose => {
+                    self.send_action("transpose");
+                }
+                EditViewCommands::AddCursorAbove => {
+                    self.add_cursor_above();
+                }
+                EditViewCommands::AddCursorBelow => {
+                    self.add_cursor_below();
+                }
+                EditViewCommands::SingleSelection => {
+                    self.single_selection();
+                }
+                EditViewCommands::SelectAll => {
+                    self.select_all();
+                }
+            }
+            ctx.invalidate();
+        }
+        true
+    }
+
+    fn key(&mut self, event: &KeyEvent, _ctx: &mut HandlerCtx) -> bool {
+        match event.key {
+            KeyVariant::Vkey(vk) => {
+                self.keydown(vk, event.mods);
+            }
+            KeyVariant::Char(ch) => {
+                self.char(ch as u32, event.mods);
+            }
+        } 
+        true
+    }
+
+    fn anim_frame(&mut self, _interval: u64, _ctx: &mut HandlerCtx) {}
```

Comment:
Again probably best to delete this for now, though there are a couple reasons we might fill it in later (smooth cursor blinking and smooth scrolling).